python/mxnet/metric.py

```diff
@@ -917,6 +917,73 @@ def update(self, labels, preds):
             self.sum_metric += (-numpy.log(prob + self.eps)).sum()
             self.num_inst += label.shape[0]
 
+@register
+@alias('log_loss')
+class LogarithmicLoss(EvalMetric):
+    """Computes logarithmic loss.
+
+    The cross entropy over a batch of sample size :math:`N` is given by
+
+    .. math::
+       -\\sum_{n=1}^{N}\\sum_{k=1}^{K}t_{nk}\\log (y_{nk}),
+
+    where :math:`t_{nk}=1` if and only if sample :math:`n` belongs to class :math:`k`.
```

Comment:
what's t and y? what's K?

Reply:
I forgot to update the equation for log loss... Will update it later